common_test.go

```diff
@@ -23,6 +23,7 @@ var (
 	flagRunAuthTest  = flag.Bool("runauth", false, "Set to true to run authentication test")
 	flagCompressTest = flag.String("compressor", "", "compressor to use")
 	flagTimeout      = flag.Duration("gocql.timeout", 5*time.Second, "sets the connection `timeout` for all operations")
+	flagSkipLong     = flag.Bool("skipLong", false, "Don't run long tests")
```

Comment:
go's testing framework already includes support for this, `go test -short` and then in the test `testing.Short()`, see https://golang.org/pkg/testing/#Short

Reply:
Removed.